test/system/001-basic.bats

```diff
@@ -10,16 +10,18 @@ function setup() {
     :
 }
 
-@test "podman --context emits reasonable output" {
-    run_podman 125 --context=swarm version
-    is "$output" "Error: Podman does not support swarm, the only --context value allowed is \"default\"" "--context=default or fail"
-
-    run_podman --context=default version
-}
```

Comment:
I simply copied this below, adding a few comments and reordering. Reason: I want 'podman version emits reasonable output' to be the very absolute first thing we test.